<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>APP update :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Core update :</string>
   </property>
  </widget>
  <widget class="QLabel" name="app_update_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>checking...</string>
   </property>
  </widget>
  <widget class="QLabel" name="core_update_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>checking...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>280</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="update_app_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Update APP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_core_button">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>190</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Core</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
